src/SortableTrait.php

```diff
@@ -65,7 +65,7 @@ public static function setNewOrder($ids, $startOrder = 1)
      *
      * @return string
      */
-    protected function determineOrderColumnName()
+    public function determineOrderColumnName()
```

Comment:
`determineOrderColumName` needed to be public in order for it to be used in the Model eventlistener